src/client/gui/lib/catalogue/launch_form.dart

```diff
@@ -83,6 +83,77 @@ class _LaunchFormState extends ConsumerState<LaunchForm> {
       width: 360,
     );
 
+    final zonesAsync = ref.watch(zonesProvider);
+    final zoneDropdown = FormField<String>(
+      initialValue: 'auto',
+      onSaved: (value) =>
+          launchRequest.zone = value == 'auto' ? '' : value ?? '',
+      builder: (field) {
+        return Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(
+              children: [
+                const Text('Pseudo zone'),
+                const SizedBox(width: 4),
+                Tooltip(
+                  message:
+                      'Pseudo zones simulate availability zones in public clouds\nand allow for testing resilience against real-world incidents',
+                  child: Icon(Icons.info_outline,
+                      size: 16, color: Colors.blue[700]),
+                ),
+              ],
+            ),
+            const SizedBox(height: 8),
+            zonesAsync.when(
+              loading: () => const SizedBox(
+                width: 20,
+                height: 20,
+                child: CircularProgressIndicator(),
+              ),
+              error: (_, __) => const Text('Error loading zones'),
+              data: (zones) {
+                final availableZones = zones.where((z) => z.available).toList();
+                final hasAvailableZones = availableZones.isNotEmpty;
+
+                return Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    DropdownButton<String>(
```

Comment:
You could use our own `Dropdown` widget, which has made to have the right styling.